documentation/README.md

```diff
@@ -0,0 +1,98 @@
+# H2O LLM Studio Documentation
+
+- The LLM Studio documentation is built using [Makersaurus](https://github.com/h2oai/makersaurus/pkgs/npm/makersaurus) which is a very thin wrapper around Facebook's Docusaurus.
+- The documentation is displayed at {{ https://docs.h2o.ai/h2o-llm-studio/ }}
+
+To view, edit, and cut a version of the documentation, the following is required:
+
+- Node.js version 16.14+ (you can check your version by running `node -v`). Use nvm to manage multiple Node versions installed on a single machine.
+
+- To install Node.js and npm with nvm in Mac or Ubuntu, run: `curl -o-
+https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash` and `nvm install node`
+
+- Makersaurus (the H2O themed documentation site) is hosted on H2O's Github npm registry. npm must authenticate to the registry before you can download Makersaurus. Follow the 3 steps below to authenticate the npm package.
+
+ If you have already installed `@h2oai/ui-kit` or any other private `@h2oai`-prefixed npm package you can skip this step.
+
+ **Step 1:** Create a "classic" [personal access token](https://github.com/settings/tokens) (PAT) on Github
```

Comment:
specify which rights are needed please

Reply:
No specific rights are needed here. I tested and verified all the steps in the documentation readme with a brand new non-h2o account and it worked fine.

Reply:
Yes, please specify that only read packages is needed:
![image](https://github.com/h2oai/h2o-llmstudio/assets/1069138/b02e1351-7209-43ad-84e2-63398479b1cf)